python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING
+
+from triton.experimental.gluon.language import _core as ttgl
+from triton.experimental.gluon.language._core import builtin
+from triton._C.libtriton import ir
+
+if TYPE_CHECKING:
+    from ..._semantic import GluonSemantic
+
+__all__ = ["buffer_load", "buffer_store"]
+
+
+@builtin
+def buffer_load(ptr, offsets, mask=None, other=None, stride=None, cache=None, _semantic=None):
+    """
+      This op is lowered into BufferLoadOp of TritonAMDGPU directly.
+      It reads from global memory and return a tensor of data.
+
+      Params:
+      ptr: A scalar base pointer, NOT a block of pointers.
+      offsets: A tensor of offsets in the global memory.
+      stride: (Optional) address difference between the first elements of each row in bytes.
+      other: (Optional) Similiar with mask in a normal load.
```

Comment:
You want to place `other` to be after `mask` given that 1) that's the order of their appearance in the parameter list and 2) you effectively explained what `other` means in `mask`..